components/backline/habitat/plan.sh

```diff
@@ -1,6 +1,6 @@
 pkg_name=hab-backline
 pkg_origin=core
-pkg_version=$(cat "$SRC_PATH/../../VERSION")
+pkg_version=$(cat "$SRC_PATH/../../VERSION_FAKE")
```

Comment:
These are so we can iterate on a fake version file and not damage the real VERSION file

Reply:
This is an outdated pattern. You'll want to use the update_version pattern. You can also allow yourselves to build the "real" version without reverting. 

```
pkg_version() {
  if [[ "<some environment variable>" == "" ]]; then 
    cat "$SRC_PATH/../../VERSION"
  else
    cat "$SRC_PATH/../../VERSION_FAKE"
  fi
}
 
do_before() {
  do_default_before
  update_pkg_version
}
```